Makefile

```diff
@@ -15,3 +15,45 @@ apply-copyright:
 fix-licenses: apply-copyright
 
 check-licenses: copyright-check
+
+.PHONY: build-linter-image lint-submodules check-linter-image check-dependencies lint-all mypy
+
+IMAGE := drx-idp:latest
+GIT_COMMIT := $(shell git log -1 --format=%H)
+
+# Build linter image
+build-linter-image:
+	docker build -t $(IMAGE) -f docker/linter/Dockerfile .
+
+check-dependencies:
+	@echo "🎱 Checking dependencies..."
+	pip install -r requirements.txt -r dev-requirements.txt -q
+	@echo "🎱 Dependencies are installed"
+
+check-linter-image:
+	@echo "🎱 Checking if linter image exists..."
+	@if DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $(IMAGE) > /dev/null; then \
```

Comment:
This line throws an error on my machine
```
🎱 Checking if linter image exists...
errors:
denied: requested access to the resource is denied
unauthorized: authentication required
```

replacing with ` docker inspect $(IMAGE)` works, but no idea if this retains the intended behaviour

Reply:
the setup: windows 11 + wsl with docker desktop

Reply:
Thanks that's resolved